### PR TITLE
[OP-270] Task. Develop a solution for displaying currency on e-commerce in UAH.

### PR DESCRIPTION
**Pull request by mykola-s**
**Description:** Develop a solution for displaying currency on e-commerce in UAH:
- Added currency field to Site configuration.
- Added OSCAR_CURRENCY_FORMAT flag to ecommerce/settings/_oscar.py
```
OSCAR_CURRENCY_FORMAT = {
    'USD': {
        'currency_digits': True,
        'format_type': "accounting",
        'format': u'\u00A4#,##0.00'
    },
    'UAH': {
        'currency_digits': True,
        'format_type': "standard",
        'format': u'\u00A4#,##0.00',
    },
}
```
- Added endpoint api/v2/currency by GET returns currency name from site.siteconfiguration.currency.
- Replaced currency in templates and code with the value in the site.siteconfiguration.currency field.

**Youtrack:** https://youtrack.raccoongang.com/issue/OP-270

**Reviewers:**
- [x] @ihor-romaniuk 
- [ ] @oksana-slu

### CODE REVIEW

File: ecommerce/static/js/views/course_detail_view.js [hunk 1]
```diff
@@ -68,7 +71,9 @@ define([
                             seat.set('credit_provider_display_name', creditProvider.get('display_name'));
                         }
                     });
-                    html += _.template(CourseCreditSeatsTemplate)({creditSeats: seats.filtered, moment: moment});
+                    html += _.template(CourseCreditSeatsTemplate)({
+                        creditSeats: seats.filtered, moment: moment, currency: this.currency
```
Comment (ihor-romaniuk) [inline ecommerce/static/js/views/course_detail_view.js]: If you have two or more properties in the object, you should write them one under one

Comment (mykola-s) [reply ecommerce/static/js/views/course_detail_view.js]: Ok. I fixed it.

File: ecommerce/extensions/api/serializers.py [hunk 2]
```diff
@@ -773,3 +774,7 @@ class ProviderSerializer(serializers.Serializer):  # pylint: disable=abstract-me
     status_url = serializers.CharField()
     thumbnail_url = serializers.CharField()
     url = serializers.CharField()
+    
+
+class CurrencySerializer(serializers.Serializer):
+    currency = serializers.CharField(max_length=3)
```
Comment (oksana-slu) [inline ecommerce/extensions/api/serializers.py]: убрать дублируется

File: ecommerce/rg_extensions/currency/api/v2/views/currency.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+from django.conf import settings
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from ecommerce.extensions.api.serializers import CurrencySerializer
+from ecommerce.rg_extensions.currency.utils import get_currency
+
+
+class CurrencyAPIView(APIView):
+    serializer_class = CurrencySerializer
```
Comment (oksana-slu) [inline ecommerce/rg_extensions/currency/api/v2/views/currency.py]: проверить используется ли вообще CurrencySerializer

File: ecommerce/templates/oscar/dashboard/offers/offer_detail.html [hunk 4]
```diff
@@ -40,7 +40,7 @@ <h1>{{ offer.name }}</h1>
             <span class="label label-success">{% trans "Offer currently available" %}</span> {% else %}
             <span class="label label-danger">{% trans "Offer not available due to restrictions!" %}</span> {% endif %}
         </td>
-        <td>{% trans "Total cost:" %} <strong>{{ offer.total_discount|currency }}</strong></td>
+        <td>{% trans "Total cost:" %} <strong>{{ offer.total_discount|currency:request.site.siteconfiguration.currency.name }}</strong></td>
```
Comment (oksana-slu) [inline ecommerce/templates/oscar/dashboard/offers/offer_detail.html:43]: проверить site.siteconfiguration.currency должна быть ошибка 
если нет объекта currency

Comment (oksana-slu) [reply ecommerce/templates/oscar/dashboard/offers/offer_detail.html:43]: запишем в загадки))
ошибки нет

**Review by oksana-slu — APPROVED** [review]

**Pull request merged**